content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+{{< glossary_definition term_id="eviction" length="all" >}}
+
+During an eviction, the kubelet sets the `PodPhase` for the selected pods to `Failed`.
+This gracefully terminates the pods. 
+
+If the pods are managed by a workload resource (such as {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+or {{< glossary_tooltip text="Deployment" term_id="deployment" >}}) that
+replaces failed pods, the control plane creates new pods in place of the evicted pods.
+
+{{<note>}}
+The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
+before it terminates end-user pods.
+{{</note>}}
+
+When evicting pods, the kubelet respects your configured [PodDisruptionBudget](/docs/tasks/run-application/configure-pdb/).
```

Comment:
This is false.  It does not respect PDBs in out-of-resource scenarios.  When using the [eviction API](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#eviction-api), PDB is respected.  But it is not respected during kubelet evictions.

Reply:
So there's two main ways in which eviction is triggered - out-of-resource scenarios and the eviction API/draining a node? Would those fall under automated and manual eviction respectively?

Reply:
Yes, but i'm trying to stress that "eviction" isn't one thing in kubernetes.  Its two completely separate things that were unfortunately given the same name.  But they share no characteristics except the name, and the fact that pods stop running where they are currently running.

Reply:
Okay. This topic will be the perfect spot to explain the distinction between the two types then, and to stress that they shouldn't and can't be interchangeable. I'll add content that tries to do that and request a review when I do!